aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -25,11 +25,28 @@ In this tutorial, you learn how to:
 
 ## Prerequisites
 
-* [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download/all)
-* [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)
-* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.7.3 or later with the following workloads:
+# [Visual Studio](#tab/visual-studio)
+
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.9 or later with the following workloads:
   * **.NET Core cross-platform development**
```

Comment:
x-plat is automatic now - right?

Reply:
What do you mean by "automatic"?

Reply:
Automatically added. You don't need to add x-plat. Adding asp.net adds xplat.

Reply:
In this case, everything works without the x-plat workload. I'll submit another PR to address it.